objects/obj_star/Alarm_1.gml

```diff
@@ -325,7 +325,63 @@ if (i=1) and (planets>0){
     if (instance_exists(obj_p_fleet)){if (point_distance(x,y,aa.x,aa.y)>50) then nostart=true;}
     if (!instance_exists(obj_p_fleet)) then nostart=true;
     
-    if (p_feature[i]="") and (p_owner[i]!=1) and (nostart=true) then p_feature[i]="????|";
+    if (array_length(p_feature[i])=0) and (p_owner[i]!=1) and (nostart=true){
+		var ranb;ranb=0;
+		// if (ranb=1) and (p_owner[i]!=1) and (p_owner[i]!=2) and (p_owner[i]!=3) then ranb=floor(random(4))+2;
```

Comment:
do you plan to keep this if?

Reply:
actually this looks like an error will check over